src/extension/devtools/devtools.ts

```diff
@@ -61,32 +85,6 @@ clientPort.on("clientNotFound", () => {
   devtoolsMachine.send("clientNotFound");
 });
 
-devtoolsMachine.subscribe(({ value }) => {
-  switch (value) {
```

Comment:
This was the main thing I was trying to get away from with this PR 🙂. `switch` has its place, but this could get unwieldy as we add more states (in fact I have another state or two in mind already).